### PR TITLE
Social logos on Homepage footer

### LINKED ISSUE #3905: Add Github, Twitter + Slack Logos to the homepage footer, center align logos + text
## Expected Behavior
The links at the bottom of the page to Github, Twitter + Slack should include logos like on the old site as well as center aligning the content in each of the boxes. 

i.e. Screen Shot 2021-07-01 at 11 40 33 AM

## Actual Behavior
Screen Shot 2021-07-01 at 11 42 25 AM

## Steps to Reproduce the Problem

1. Visit the homepage
1. Scroll down

### PR DESCRIPTION
**Pull request by cardil**
Fixes #3905

## Proposed Changes

- Adding logos for social links on Homepage footer

### CODE REVIEW

File: overrides/assets/stylesheets/home.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import url('//cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css');
```
Comment (cardil) [inline overrides/assets/stylesheets/home.css:1]: Font Awesome was used on old site, but IDK is this okay to add it here.

Comment (julz) [inline overrides/assets/stylesheets/home.css:1]: do we need this? I think the theme already bundles font-awesome (https://squidfunk.github.io/mkdocs-material/reference/icons-emojis)

Comment (julz) [reply overrides/assets/stylesheets/home.css:1]: see https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#using-icons-in-templates for how to use this in a template like home.html

Comment (cardil) [reply overrides/assets/stylesheets/home.css]: Removed.

Comment (cardil) [reply overrides/assets/stylesheets/home.css]: Okay. I've migrated to use native mkdocs icons.

**Comment by kvmware** [comment]
Looks good :+1:

**Pull request merged**